File: utils/VectorEnginePlugin.ref/pares/funciones/primCurrentMorphId.js
function primCurrentMorphId(argCount) {
    var aNumber;
    if (!isIntegerObject((aNumber = stackValue(0)))) {
        primitiveFailFor(PrimErrBadArgument); return false;
    }
    currentMorphId = aNumber >>> 0;
    if (!failed()) pop(1);
    return !failed();
}

File: utils/VectorEnginePlugin.ref/pares/funciones/primCurrentMorphId.c
/* VectorEnginePlugin>>#currentMorphId: */
EXPORT(sqInt)
primCurrentMorphId(void)
{
	sqInt aNumber;

	if (!(isIntegerObject((aNumber = stackValue(0))))) {
		return primitiveFailFor(PrimErrBadArgument);
	}
	aNumber = integerValueOf(aNumber);
	currentMorphId = aNumber;
	if (!(failed())) {
		pop(1);
	}
	return null;
}
